benchmark/citation/appnp.py

```diff
@@ -5,7 +5,8 @@
 from citation import get_planetoid_dataset, random_planetoid_splits, run
 from torch.nn import Linear
 
-from torch_geometric.nn import APPNP
+from torch_geometric.nn import APPNP as Conv
```

Comment:
Any reason for this change?

Reply:
Just to reuse `Conv.__name__` in `rename_profile_file` in various conv layers.